Cargo.toml

```diff
@@ -22,6 +22,8 @@ rusb = "0.9"
 thiserror = "1.0.40"
 qrcode =  { version = "0.12", optional = true }
 log = "0.4"
+rayon = "=1.7.0"
```

Comment:
Are these being used? Or are these left over from something else you were trying?

Reply:
Yeah I was trying something that pulled this dependency in with the latest not compatible with the version of rust we're using, so I pinned this one. Reverted.

Reply:
If you update https://github.com/flynnguy/posify/blob/master/rust-toolchain can you unpin it?

Reply:
Yeah, I'll delete rust-toolchain, I don't think we need it anymore